source/MaterialXFormat/Util.cpp

```diff
@@ -222,4 +223,26 @@ FileSearchPath getSourceSearchPath(ConstDocumentPtr doc)
     return searchPath;
 }
 
+FileSearchPath getDefaultDataSearchPath(const FilePath& startPath)
```

Comment:
@kwokcb This looks really close to the mark, though I'd recommend omitting the unused `startPath` argument from this helper function, as it's much easier to add new arguments if needed later on than it is to remove ones that may never have been used.  We may ultimately find that some other degree of freedom is needed by clients, e.g. controlling the input library folder name rather than the starting folder, and it will be difficult to guess these needs before they arise.

Reply:
Sounds good. I'll do the cleanup